stores/reducers.js

```diff
@@ -1,22 +1,28 @@
 import { combineReducers } from 'redux';
 import { isBrowser } from '../server/utils/browserUtil';
 
-function subject(state = isBrowser() ? window.SUBJECT : null, action) {
```

Comment:
Why are `action` ommitted?

Reply:
They are unused, so I removed them to reduce confusion.